app/build.gradle.kts

```diff
@@ -42,6 +42,10 @@ android {
 			excludes += "/META-INF/{AL2.0,LGPL2.1}"
 		}
 	}
+	compileOptions {
+		sourceCompatibility = JavaVersion.VERSION_17
+		targetCompatibility = JavaVersion.VERSION_17
+	}
```

Comment:
Try to apply the same settings for lint module

Reply:
Actually, it is not a problem with a Gradle configuration – locally everything is working fine. The problem is with our Github Actions configuration, the setup java is wrong configured so the build action is using the default Java from the Linux distro (which is 11, that's why it was working before 😄).

Reply:
@MarekMacko Now it works! The problem lay elsewhere than we expected 😄

Reply:
Always expect unexpected 😄